### DISCUSSION TITLE
App VS Pages when no routing is needed

### DISCUSSION THREAD

**Discussion by azohc** [vercel/next.js#63923] [Help]
### Summary

Hi! I'm building a website that will essentially be a long landing page that ends with a CTA.
There will be no need for routing, but I'm going with Next because of the DX and other benefits of this wonderful framework.

My question, with the opt-out of routing in mind, is whether I should use the app directory or the pages directory.
Are there any other things that I should look out for if I don't plan to use routing?

Thanks in advance!

**Comment by mutasim77** [comment]
As long as your project is a small, static page, I would recommend using the `pages` directory. It's the perfect choice for this scenario.  However, it depends on your preference; the `app` directory can also be used.

P.S. Since you've chosen NextJS for your landing page instead of plain HTML/CSS, remember to take advantage of its features like `SSG` to improve performance.

All the best!
